wren-ui/src/apollo/server/mdl/mdlBuilder.ts

```diff
@@ -118,46 +119,114 @@ export class MDLBuilder implements IMDLBuilder {
     });
   }
 
-  public addColumn(): void {
+  public addNormalField(): void {
```

Comment:
Good naming change to reflect the difference between columns and calculated fields.